package.json

```diff
@@ -184,5 +184,8 @@
   },
   "publishConfig": {
     "registry": "https://npm.pkg.github.com/"
+  },
+  "dependencies": {
```

Comment:
why do we need it there as a dependency too?

Reply:
I don't have an answer for you, was added by yarn 🤷🏻‍♂️